draft-regext-rdap-x-media-type.md

```diff
@@ -139,8 +134,19 @@ passed from the client to the server, and the presence of these protocol element
 used by servers to determine a client's capability to handle the RDAP extension. This specification
 does not require the usage of those extension identifiers in the "extensions" parameter,
```

Comment:
Are those protocol elements actually extension identifiers? Perhaps better to say "related extension identifiers"?